solr/core/src/test/org/apache/solr/handler/admin/api/V2CollectionAPIMappingTest.java

```diff
@@ -85,6 +85,18 @@ public void testGetCollectionStatus() throws Exception {
     assertEquals("shard2", v1Params.get(SHARD));
   }
 
+
+  @Test
+  public void testRenameCollectionAllParams() throws Exception {
+    final SolrParams v1Params = captureConvertedV1Params(
+            "/collections/collName", "POST", "{\"rename\": {\"to\": \"targetColl\"}}");
```

Comment:
The first thing that jumps out at me in this test code is that you restructured the API endpoint to look the way we discussed.  (i.e. now it looks like `POST /collections/collName/commands/rename`)....but this test still uses the old appearance/syntax.  The path, request-body, etc. that gets sent into `captureConvertedV1Params` is all using the old syntax we moved away from.  If you fix that, I'm betting this error will go away!

As a general note on "process": generally speaking its easier for me (or others) to help review if you push up the code changes you've made, even if things aren't 100% working yet.  That makes it easier for me to comment on particular lines.  It makes it easier for me to pull the code down and run that test myself, etc.  It saves you from having to paste your code in a comment box and fight the markdown/formatting, etc.

Totally fine if you had reasons for not doing that here; just figured I'd mention the convention as a "heads up" in case it's helpful since you're newer to the community